mkdocs/docs/HPC/index.md

```diff
@@ -1,10 +1,30 @@
-# Welcome
+---
+hide: "toc"
+---
+#  Welcome to the HPC-Ugent documentation
 
-Welcome to the {{ hpcinfra }} documentation!
+---
+
+
+![image](img/ch0-hpc.png){: align=left style="height:383px;width:378px;border-radius:6px" }
 
 Use the menu on the left to navigate, or use the search box on the top right.
 
-The current view of the documentation is intended for people using **{{ OS }}** as operating system.
-To switch to different operating system (Windows, macOS, Linux), click [here](../#force_select_OS).
+You are viewing documentation intended for people using **{{ OS }}**. 
+Use the OS dropdown in the top bar to switch to a different operating system.
```

Comment:
Maybe also force a line break here, will look a lot cleaner wit normal browser window width imho:

```suggestion
You are viewing documentation intended for people using **{{ OS }}**.<br/>
Use the OS dropdown in the top bar to switch to a different operating system.

Reply:
stop with teh br. double newline is what markdown prefers

Reply:
Double newline starts a new paragraph, here you want to keep these two sentences together in the same paragraph.

But I guess you mean a double space at the end of the line, that's also a hard line break in MarkDown, see https://stackoverflow.com/questions/36583502/how-to-force-a-linebreak.

So, new suggestion:

```suggestion
You are viewing documentation intended for people using **{{ OS }}**.  
Use the OS dropdown in the top bar to switch to a different operating system.

Reply:
no, i was mistaken. this is worse, as trailing whitespace is just annoying.  stick to the html tags if you really must use linebreak.

Reply:
I fully agree that the double space is horrible, so `<br/>` it is then (it's cosmetic, but I consider that important for a landing page especially).